index.html

```diff
@@ -4,47 +4,185 @@
+                        <img src="assets/dot.png" alt="" />
+                        <h4>2015</h4>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="styled-links">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="assets/tonic-bg-2.png" alt=""/>
+                </div>
+                <div class="card-title">
+                    <h2>Tonic</h2>
+                    <div class="caption">
+                        <h3>CANOPY </h3>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>BackEndDev</h4>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>2015</h4>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="styled-links">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="card">
+                <div>
+                    <img src="assets/mps-bg-2.png" alt=""/>
+                </div>
+                <div class="card-title">
+                    <h2>
+                        Multi-Post <br />
+                        Stories
+                    </h2>
+                    <div class="caption">
+                        <h3>CANOPY </h3>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>BackEndDev</h4>
+                        <img src="assets/dot.png" alt="" />
+                        <h4>2015</h4>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="styled-links">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                </ul>
+                <ul class="btn">
+                    <li><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="profile-card">
+                <div class="header-text">
+                    <img src="assets/me.png" alt=""/>
```

Comment:
- [ ] The About Myself text is supposed to be a heading with the `h1`, `h2, `h3`... tag but not an image. Please change this 👍